src/ExpectedMark.js

```diff
@@ -47,21 +55,7 @@ ExpectedMark.prototype.onComplete = function(onMark) {
  * See Nolan Lawson's article describing the issue and proposing this method:
  * https://nolanlawson.com/2018/09/25/accurately-measuring-layout-on-the-web/
  */
-ExpectedMark.prototype.waitForNextPaintAndRecord = function() {
-	// In development mode, include DEBUG timestamps to show when
-	// original calls were fired to see the impact
-	if (
-		process.env.NODE_ENV !== 'production' &&
-		typeof window.console !== 'undefined' &&
-		typeof window.console.timeStamp !== 'undefined'
-	) {
-		window.console.timeStamp(`[DEBUG] original call for ${this.props.name}`);
```

Comment:
I've removed this debug info because I think we've verified it works as intended